src/jit.c

```diff
@@ -1336,6 +1336,25 @@ static void ra_load2_sext(struct jit_state *state,
     }
 }
 
+void parse_branch_history_table(struct jit_state *state, rv_insn_t *ir)
+{
+    int max_idx = 0;
+    for (int i = 0; i < HISTORY_SIZE; i++) {
```

Comment:
Declare a variable which accesses `ir->branch_table`, so that you can write shorter code.